filebeat/tests/system/test_autodiscover.py

```diff
@@ -11,6 +12,8 @@ class TestAutodiscover(filebeat.BaseTest):
     """
     Test filebeat autodiscover
     """
+    # 1/20 build fails https://github.com/elastic/beats/issues/21305
+    @pytest.mark.flaky(reruns=1, reruns_delay=10)
```

Comment:
Autodiscover tests should be fixed after https://github.com/elastic/beats/pull/21242

Reply:
removed from the PR